src/EPR.PRN.Backend.API/Controllers/PrnController.cs

```diff
@@ -37,22 +37,22 @@ public async Task<IActionResult> GetPrn([FromHeader(Name = "X-EPR-ORGANISATION")
             return Ok(prn);
         }
 
-		[HttpGet("v1/search/{page?}/{search?}/{filterBy?}/{sortBy?}")]
-		[ProducesResponseType(typeof(PaginatedResponseDto<PrnDto>), 200)]
-		[ProducesResponseType(400)]
-		[ProducesResponseType(401)]
-		public async Task<IActionResult> GetSearchPrns([FromHeader(Name = "X-EPR-ORGANISATION")] Guid orgId,
+        [HttpGet("v1/search/{page?}/{search?}/{filterBy?}/{sortBy?}")]
+        [ProducesResponseType(typeof(PaginatedResponseDto<PrnDto>), 200)]
+        [ProducesResponseType(400)]
+        [ProducesResponseType(401)]
+        public async Task<IActionResult> GetSearchPrns([FromHeader(Name = "X-EPR-ORGANISATION")] Guid orgId,
             [FromQuery] PaginatedRequestDto request)
-		{
+        {
             if (orgId == Guid.Empty)
                 return Unauthorized();
 
             var result = await _prnService.GetSearchPrnsForOrganisation(orgId, request);
-            
+
             return Ok(result);
-		}
+        }
```

Comment:
Please remove these changes related to PRN 1 's code. I think its a bit a confusing.

Reply:
this is just auto formatting there isnt any code change. Think visual studio does it on save. Maybe more of a hassle to revert if the formatting is correct?